### DISCUSSION TITLE
Allow Chinese language pack - SEB 3.5.0 - Win-IOS

### DISCUSSION THREAD

**Discussion by aidandauphin** [SafeExamBrowser/seb-win-refactoring#650] [Q&A]
We want to allow students to use a Chinese language pack in their exam. 

Windows settings:
1. kiosk mode "Disable Explorer Shell"
    a. Attempted exam in SEB, the user is able to type in Chinese characters, but not getting the conversion suggestion pop-up box.
2. Permitted Process for ChsIME.exe
   a. Attempted exam in SEB, the user is able to type in Chinese characters, but not getting the conversion suggestion pop-up box.

Same out come in MAC OSX ie No pop up conversion box

 
Found previous support tickets relating to this but these were 2021.
Link:: https://sourceforge.net/p/seb/discussion/844844/thread/96379b3c/#cd5c

Assistance with this would be appreciated

**Comment by danschlet** [comment]
In the Mac repository, a user confirmed that he was able to use Chinese character input in macOS (by using the AAC Assessment Mode instead of the default SEB lockdown mode) and even on Windows: https://github.com/SafeExamBrowser/seb-mac/issues/251. On Windows I assume you need to use "Disable Explorer Shell" and enable the "Show Keyboard" option. We should try to confirm this, then we could add it to the SEB manuals.
